test/truefi2/TrueCreditAgency.test.ts

```diff
@@ -256,11 +259,11 @@ describe('TrueCreditAgency', () => {
     beforeEach(async () => {
       pool1 = await deployMockContract(owner, TrueFiPool2Json.abi)
       pool2 = await deployMockContract(owner, TrueFiPool2Json.abi)
-      await pool1.mock.poolValue.returns(parseEth(10))
+      await pool1.mock.poolValue.returns(parseEth(10_000))
```

Comment:
I really do not like using mock contracts in test suites, especially those that are developed by us. I know, that it was not introduced with this PR. I probably missed it in one of the old ones. Consider changing it here. It does not look to be too big of a change for me, but if any complications may occur, we can still do it in a new PR.